liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -283,6 +286,26 @@ private Liquibase createLiquibase(String changeLogFile, ResourceAccessor resourc
         return new Liquibase(changeLogFile, resourceAccessor, database);
     }
 
+    @Test
+    public void testEmptyRollbackableSqlChangeLog() throws Exception {
+        assumeNotNull(this.getDatabase());
+
+        Liquibase liquibase = createLiquibase(emptyRollbackSqlChangeLog);
+        clearDatabase();
+
+        liquibase = createLiquibase(emptyRollbackSqlChangeLog);
+        liquibase.update(this.contexts);
+
+        liquibase = createLiquibase(emptyRollbackSqlChangeLog);
+        liquibase.rollback(new Date(0), this.contexts);
+
+        liquibase = createLiquibase(emptyRollbackSqlChangeLog);
```

Comment:
The update and rollback at line 302 and line 305 are duplicates of the update and rollback at lines 299 and 302. Did you mean to use a different changelog?

Reply:
No, sorry. It is duplicated code, I'm removing it. thanks for spot that.